audio/out/ao_oss.c

```diff
@@ -49,6 +49,9 @@
 #endif
 
 #define PATH_DEV_DSP "/dev/dsp"
+#ifndef PATH_DEV_MIXER
+#define PATH_DEV_MIXER "/dev/mixer"
+#endif
```

Comment:
As discussed this should unconditionally define `PATH_DEV_MIXER` and the commit message should be more into the direction of "add missing define"

Reply:
Yup. Was kind of funny how I had this feeling at the back of my head that this was an internal define that was forgotten, but then after some checking another person also ended up with the same result :) .

Finally got to it, and done.

(technically for things like this I'd prefer `static const char *` or so, but eh, this fixes the build issue :) )